guides/V1_wwsg_Webhooks.md

```diff
@@ -33,6 +33,8 @@ Available events are:
 
 **message-deleted**: each time a message is deleted from the space. Please note that the **message-deleted** event API is currently `experimental` and subject to change. It is being included here to engage developers early with this API.
 
+**message-edited**: each time message content is edited in the space.  Please note that the triggering of **message-edited** events can only result from administrator actions and not from end users.
```

Comment:
This part perhaps could be adjusted a bit or removed altogether?:

>Please note that the triggering of **message-edited** events can only result from administrator actions and not from end users.

To:

"Please note that the triggering of **message-edited** events takes place due to a change in the data model  via Administrative, End user or programatic means.  Currently only Administrative means are implemented."

Reply:
Addressed in commit 76554ff